src/gltf-component-mappings.js

```diff
@@ -13,12 +13,30 @@ AFRAME.GLTFModelPlus.registerComponent("gltf-model-plus", "gltf-model-plus");
 AFRAME.GLTFModelPlus.registerComponent("media-loader", "media-loader");
 AFRAME.GLTFModelPlus.registerComponent("body", "body");
 AFRAME.GLTFModelPlus.registerComponent("hide-when-quality", "hide-when-quality");
-AFRAME.GLTFModelPlus.registerComponent("light", "light");
+AFRAME.GLTFModelPlus.registerComponent("light", "light", (el, componentName, componentData) => {
+  if (componentData.distance === 0) {
+    componentData.decay = 0;
+  }
+  el.setAttribute(componentName, componentData);
+  if (componentData.castShadow) {
+    // HACK: component.light isn't set until one frame after setArrtibute
```

Comment:
:( this should be handled in the light component, but since its a pain to fix it and since this is going to go away once we move over our default scenes I guess its ok.

Reply:
Yeah, I think we should just remove the `light` component from the gltf-component-mappings as soon as our ported environments land.